package.json

```diff
@@ -36,7 +36,7 @@
     "markdown-it-sanitizer": "~0.3.0",
     "markdown-it-toc": "~1.1.0",
     "markdown-it-video": "~0.2.1",
-    "mithril": "^0.1.30",
+    "mithril": "^0.2",
```

Comment:
Can you add `mithril` to the vendor bundle in the webpack config so we don't load mithril multiple times on the same page?